sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1680,6 +1681,13 @@ public partial class PiiDetectionSkill : Azure.Search.Documents.Indexes.Models.S
         public static bool operator !=(Azure.Search.Documents.Indexes.Models.PiiDetectionSkillMaskingMode left, Azure.Search.Documents.Indexes.Models.PiiDetectionSkillMaskingMode right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class PrioritizedFields
+    {
+        public PrioritizedFields() { }
+        public System.Collections.Generic.IList<Azure.Search.Documents.Indexes.Models.SemanticField> ContentFields { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.Search.Documents.Indexes.Models.SemanticField> KeywordsFields { get { throw null; } }
```

Comment:
Should this be `KeywordsFields` or `KeywordFields`?  The latter reads better to me.

Reply:
The plural here is because we want to make it clear that a single string doesn't have to be a single keyword. For example, a field containing a full URL would be a "keywordsField" as it can be interpreted as each part of the URL is a different keyword (example: "github.com/Azure/azure-sdk-for-net contain" is made of multiple keywords).

Reply:
I am inclined to keep it as it now is. I think that intent still comes across with `KeywordFields`. The pluralization of `keywords` would not explain much to the user who is not aware of this fact (that a single value can contain multiple keyword tokens).

Reply:
I can go either way, but let's dig into this more for GA.